src/handlers/payout/post.ts

```diff
@@ -243,28 +244,63 @@ const generatePermitForComments = async (
  * @param incentives - The basic price table for reward calculation
  * @returns - The reward value
  */
-const calculateRewardValue = (comments: Record<string, string[]>, incentives: Incentives): Decimal => {
+const calculateRewardValue = (comments: Record<MarkdownItem, string[]>, incentives: Incentives): Decimal => {
   let sum = new Decimal(0);
-  for (const key of Object.keys(comments)) {
-    const value = comments[key];
+  for (const item of MarkdownItems) {
+    const value = comments[item];
 
     // if it's a text node calculate word count and multiply with the reward value
-    if (key == "#text") {
+    if (item === MarkdownItem.Text) {
       if (!incentives.comment.totals.word) {
         continue;
       }
       const wordReward = new Decimal(incentives.comment.totals.word);
       const reward = wordReward.mul(value.map((str) => str.trim().split(" ").length).reduce((totalWords, wordCount) => totalWords + wordCount, 0));
       sum = sum.add(reward);
     } else {
-      if (!incentives.comment.elements[key]) {
+      const htmlTag = MarkdownItemToHTMLTag[item];
+      if (!htmlTag || !incentives.comment.elements[htmlTag]) {
         continue;
       }
-      const rewardValue = new Decimal(incentives.comment.elements[key]);
+      const rewardValue = new Decimal(incentives.comment.elements[htmlTag]);
       const reward = rewardValue.mul(value.length);
       sum = sum.add(reward);
     }
   }
 
   return sum;
 };
+
+const MarkdownItemToHTMLTag: Record<MarkdownItem, string> = {
+  [MarkdownItem.Text]: HTMLItem.P,
+  [MarkdownItem.Paragraph]: HTMLItem.P,
+  [MarkdownItem.Heading]: HTMLItem.H1,
+  [MarkdownItem.Heading1]: HTMLItem.H1,
+  [MarkdownItem.Heading2]: HTMLItem.H2,
+  [MarkdownItem.Heading3]: HTMLItem.H3,
+  [MarkdownItem.Heading4]: HTMLItem.H4,
+  [MarkdownItem.Heading5]: HTMLItem.H5,
+  [MarkdownItem.Heading6]: HTMLItem.H6,
+  [MarkdownItem.ListItem]: HTMLItem.LI,
+  [MarkdownItem.List]: HTMLItem.UL,
+  [MarkdownItem.Link]: HTMLItem.A,
+  [MarkdownItem.Image]: HTMLItem.IMG,
+  [MarkdownItem.BlockQuote]: HTMLItem.BLOCKQUOTE,
+  [MarkdownItem.Code]: HTMLItem.PRE,
```

Comment:
I'm not sure if it's gonna be clear to partners that `<pre>` refers to 
```
code
```
and `<code>` refers to `inline code`